nixos/modules/flyingcircus/platform/default.nix

```diff
@@ -286,12 +278,16 @@ in
     services.openssh.enable = true;
     services.nscd.enable = true;
 
+    services.journald.extraConfig = "SystemMaxUse=1G";
     systemd.tmpfiles.rules = [
-      # D instead of R to a) respect the age rule and allow exclusion
+      # d instead of r to a) respect the age rule and b) allow exclusion
       # of fc-data to avoid killing the seeded ENC upon boot.
-      "d /tmp - - - 3d"
+      "d /tmp 1777 root root 3d"
       "x /tmp/fc-data/*"
-      "D /var/tmp - - - 7d"];
+      "d /var/tmp 1777 root root 7d"
+      "d /srv"
+      "z /srv 0755 root root"
```

Comment:
Verified that this works on reboot. However, "d" rules don't show immediate effects after nixos-rebuild switch. This does not matter in the specific case (the first boot gets it right), but we must be careful with the creation of role-specific directories. Activation scripts are an better option in most cases.